cli/viam/main.go

```diff
@@ -17,6 +18,9 @@ import (
 	rdkcli "go.viam.com/rdk/cli"
 )
 
+// CliVersion set with ldflags.
+var CliVersion = "(dev)"
```

Comment:
config.Version is set by CI already, might be good to just use that if we're using the same version as rdk for now